Gemfile

```diff
@@ -1,2 +1,13 @@
 source 'https://rubygems.org'
 gemspec
+
+group :optional do
+  gem 'activerecord', '~> 4.0'
+  gem 'dbd-sqlite3'
+  gem 'dbi'
```

Comment:
This way,
- `bundle install` : Installs the gemspec gems, as well as ALL the optional parser gems.
- `bundle install --without optional` : Installs only gemspec gems.

Does this work?